src/EnergyPlus/ChillerElectricEIR.cc

```diff
@@ -1962,6 +1962,8 @@ void ElectricEIRChillerSpecs::calculate(EnergyPlusData &state, Real64 &MyLoad, b
             state, this->CWPlantLoc, this->CondMassFlowIndex, this->CDPlantLoc, DataPlant::CriteriaType::MassFlowRate, this->CondMassFlowRate);
 
         if (this->CondMassFlowRate < DataBranchAirLoopPlant::MassFlowTolerance) {
+            // Shut chiller off if there is no condenser water flow
+            MyLoad = 0.0;
```

Comment:
This does look like it boils down to this. I looked at PullCompInterconnectTrigger to see what that was trying to accomplish. This code look like it can be deleted. I'm not asking for that to happen here but it's extra clock cycles.

        // Check to make sure we didn't reuse the index in multiple components
        if (CurCriteria.CallingCompLoopNum != plantLoc.loopNum || CurCriteria.CallingCompLoopSideNum != plantLoc.loopSideNum ||
            CurCriteria.CallingCompBranchNum != plantLoc.branchNum || CurCriteria.CallingCompCompNum != plantLoc.compNum) {
            // Diagnostic fatal: component does not properly utilize unique indexing
        }